packages/cli-kit/assets/cli-ruby/lib/shopify_cli/version.rb

```diff
@@ -1,3 +1,3 @@
 module ShopifyCLI
-  VERSION = "2.34.0"
+  VERSION = ENV.fetch("SHOPIFY_CLI_VERSION", "2.34.0")
```

Comment:
Not sure about that fallback value... I prefer it to fail rather than setting a wrong value. And it shouldn't fail.
```suggestion
  VERSION = ENV.fetch("SHOPIFY_CLI_VERSION")
```

Reply:
but the CLI3 request is not failing in case the CLI_KIT_VERSION. is wrong or missing, should we implement the same strategy here right?

Reply:
CLI3 does fail when CLI_KIT_VERSION is missing. If it's wrong, I guess both will work because we don't validate it.

Reply:
And In case we want to default to something, shouldn't we default to `3.x.x`? 2.x.x will be broken no?

Reply:
@isaacroldan @gonzaloriestra Added the next CLI3 version value as fallback. That should be the first version that includes support for the new env variable